package.json

```
@@ -1,12 +1,14 @@
 {
   "name": "react_decompose",
-  "homepage": ".",
+  "homepage": "http://.",
```

```suggestion
  "homepage": ".",
```
You don't need to change it. Your Demo link is not working now. This can be caused by changes in package.json